Task/add codeowners file and organize files into folders

meman0303 opened the pull request:
Fixed all #include statements to have folder paths and tested

Made .gitignore for the repository
    will exclude all .vscode and .idea

Made Urchin.CODEOWNERS

Added Typed Python in API

Added first stage of threaded API

Relocated "reactions" , "react" and "item" python classes into Types.py
closes #15

meman0303 committed:
Moved Urchin_Firmware files into folders to organize them.
Fixed all #include statements to have folder paths

Made .gitignore for the repository
    will exclude all .vscode and .idea

Made Urchin.CODEOWNERS

meman0303 committed:
Changed "com.c" to "Com.c"
Changed "com.h" to "Com.h"

jake-molnia commented on .gitignore:
Love this exactly what I was talking about! But make sure to add like anything that (perhaps could sneak into the code) Maybe not as in depth as this but you should probably cpoy paste a template like this, and then delete the unwanted - thats how i usually do it: 
**https://gist.github.com/Yousha/3830712334ac30a90eb6041b932b68d7**

meman0303 replied:
Now I know about the Star

jake-molnia commented on Urchin.CODEOWNERS:
Noice! Exactly what I was thinking of!

meman0303 committed:
Used link Jacob gave me to add to the Gitignore

https://gist.github.com/Yousha/3830712334ac30a90eb6041b932b68d7

Also include built in READMEs for Mollusk
